[tracks] load tracks from osm url

sikmir opened the pull request:
[Public GPS traces](https://www.openstreetmap.org/traces)

Примеры валидных ссылок:
- https://www.openstreetmap.org/user/sikmir/traces/2698308

wladich commented:
Я сейчас делаю радикальный рефакторинг, там всё готово, осталось протестировать. 
Ветка tracks-parsers-refactor-to-classes, пока она не в мастере, надо отребейзится на неё.
И пожалуйста, добавьте в commit-message номер PR или issue

sikmir commented:
> Ветка tracks-parsers-refactor-to-classes, пока она не в мастере, надо отребейзится на неё.

Rebased.

> И пожалуйста, добавьте в commit-message номер PR или issue

Fixed.

wladich commented:
Для треков, у которых в файле нет тега name, получаем имя "data". Пример: https://www.openstreetmap.org/user/Mat2047135/traces/2849387

Лучше сделать, чтобы имена у разных треков различались, например типа "OSM track 324234"

sikmir commented:
> Лучше сделать, чтобы имена у разных треков различались, например типа "OSM track 324234"

Поправил.
